File: programme/radian-to-degree/radian-to-degree.cs
using System;

public class RadToDeg
{
    public static void Main(string[] args)
    {
        //Checking for valid input
        bool success = false;
        do{
            Console.Write("Input radians: ");
            try
            {
                RadianToDegree(Convert.ToDouble(Console.ReadLine()));
                success = true;
            }
            catch (System.Exception)
            {
                Console.WriteLine("Invalid Input!!");
            }
        }while(!success);

    }
    
    
    public static void RadianToDegree(double radians)
    {
        double degrees = (180 / Math.PI) * radians;
        Console.WriteLine($"{radians} Radians is {degrees} Degrees.");
    }
}

File: programme/radian-to-degree/radian-to-degree.php
<?php
// Write a programmethat converts radiant do degrees

function radiant_to_degrees($radiant)
{

    if (!is_numeric($radiant)) {

        return 'Invalid input: Input is not a number';

    }

    return round($radiant * (180 / pi()), 4);

}

$radiant = readline("Enter radiant value : ");

$degrees = radiant_to_degrees($radiant);

print "\nRadian : {$radiant}";
print "\nDegree : {$degrees}\n";
